threedi_modelchecker/checks/cross_section_definitions.py

```diff
@@ -16,7 +16,7 @@ def __init__(self, column, *args, **kwargs):
     def shape_msg(self):
         if self.shapes is None:
             return "{all}"
-        return {x.value for x in self.shapes}
+        return sorted([x.value for x in self.shapes])
```

Comment:
This modification changes the return type from set to list
```suggestion
        return set(sorted([x.value for x in self.shapes]))
```

Furthermore, the return type of this function is inconsistent. It either returns a list or a string.

Reply:
> This modification changes the return type from set to list
This is intentional. Sets are unsorted, so that leads to randomly ordered shapes in error messages.

Reply:
`sorted(set(.....))`? I don't know if the original meaning of `set` was to filter out duplicates?

Reply:
List or string: it *looks* to be only used in log messages. Returning `", ".join(.....)` could be a nice solution.

(I'll keep out of this for the rest, I don't know this repo :-) I was looking at Eli's other PR and got sidetracked.)

Reply:
> `sorted(set(.....))`? I don't know if the original meaning of `set` was to filter out duplicates?

Set is by definition unsorted. For the unique values, what could work is
sorted(list({x.value for x in self.shapes}))

Reply:
> Furthermore, the return type of this function is inconsistent. It either returns a list or a string.

oops, I'll fix that